examples/node/README.md

```diff
@@ -22,18 +22,24 @@ INSTALL
 TRY IT!
 -------
 
+There are two variants of these examples: one with code dynamically generated at runtime using Protobuf.js and one with code statically generated using `protoc`. The examples behave identically, and either server can be used with either client.
```

Comment:
Maybe add an additional sentence at the start here and restructure slightly:

There are two ways to generate the code needed to work with protocol buffers in Node.js - one approach uses Protobuf.js to dynamically generate the code at runtime, the other uses code statically generated using the protocol buffer compiler `protoc`. Because of this, there are two variants of these examples, one using dynamically generated code, one using statically generated code. The examples behave...